src/catchup/test/CatchupWorkTests.cpp

```diff
@@ -124,115 +124,141 @@ std::vector<std::pair<uint32_t, CatchupConfiguration>> gCatchupRangeCases{
     {64, {320, maxCount, CatchupConfiguration::Mode::OFFLINE_BASIC}}};
 }
 
-TEST_CASE("compute CatchupRange from CatchupConfiguration", "[catchup]")
+static void
+runExperimentalBucketListTest(std::function<void(Config const&)> f)
 {
-    VirtualClock clock;
-    auto app = createTestApplication(clock, getTestConfig());
-    auto& historyManager = app->getHistoryManager();
-
-    for (auto const& test : gCatchupRangeCases)
+    SECTION("EXPERIMENTAL_BUCKETLIST_DB enabled")
     {
-        auto lastClosedLedger = test.first;
-        auto configuration = test.second;
+        auto cfg = getTestConfig();
+        cfg.EXPERIMENTAL_BUCKETLIST_DB = true;
+        f(cfg);
+    }
 
-        auto name = fmt::format("lcl = {}, to ledger = {}, count = {}",
-                                lastClosedLedger, configuration.toLedger(),
-                                configuration.count());
-        CLOG_DEBUG(History, "Catchup configuration: {}", name);
-        {
-            auto range =
-                CatchupRange{lastClosedLedger, configuration, historyManager};
+    SECTION("EXPERIMENTAL_BUCKETLIST_DB disabled")
+    {
+        auto cfg = getTestConfig();
+        f(cfg);
+    }
+}
 
-            CLOG_DEBUG(History,
-                       "computed CatchupRange: first={}, last={}, "
-                       "replayFirst={}, replayCount={}, replayLimit={}",
-                       range.first(), range.last(), range.getReplayFirst(),
-                       range.getReplayCount(), range.getReplayLimit());
+TEST_CASE("compute CatchupRange from CatchupConfiguration", "[catchup]")
```

Comment:
Same thing here: these tests just verify that catchup configuration is correct, so it's unrelated to the flag.

Reply:
looks like this one still needs to be addressed